docs/reference/aggregation.md

```diff
@@ -0,0 +1,488 @@
+---
+title: Aggregation
+sidebar_position: 11
+---
+
+An aggregation summarizes your data as statistics on buckets or metrics.
+
+Aggregations can provide answer to questions like:
+
+- What is the average price of all sold articles?
+- How many errors with status code 500 do we have per day?
+- What is the average listing price of cars grouped by color?
+
+There are two categories: [Metrics](#metric) and [Buckets](#bucket-aggregations).
+
+#### Prerequisite
+
+To be able to use aggregations on a field, the field needs have a fast field index created. A fast field index is a columnar storage, 
+where documents values are extracted and stored to.
+
+Example to create a fast field on text for term aggregations.
+```yaml
+name: category
+type: text
+tokenizer: raw
+record: basic
+fast: true
+```
+
+See the [index config](./index-config.md) for more details and examples.
+
+#### Format
+
+The aggregation request and result de/serialize into elasticsearch compatible JSON. 
+If not documented otherwise you should be able to drop in your elasticsearch aggregation queries.
+
+In some examples below is not the full request shown, but only the payload for `aggregations`.
```

Comment:
I would put always the complete query. By the way, can't we put `"max_hits": 0` in the request?

Reply:
I updated it to always include the complete query